### PR TITLE
Add alert CDIMultipleDefaultVirtStorageClasses

### PR DESCRIPTION
**Pull request by akalenyu**
Related to https://github.com/kubevirt/containerized-data-importer/pull/2913 efforts

### CODE REVIEW

File: docs/runbooks/CDIMultipleDefaultVirtStorageClasses.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This alert fires when there are multiple default virtualization storage classes.
+
+## Impact
+
+With multiple default storage classes present, any DataVolume requesting the default storage class (not explicitly specifying a storage class name) gets the most recently created default storage class, regardless of the default status of that storage class, and the administrator receives an alert.
+
+## Diagnosis
+
+Observe multiple storage classes are marked as default for virtualization purposes:
```
Comment (apinnick) [inline docs/runbooks/CDIMultipleDefaultVirtStorageClasses.md:16]: Suggestion:

Obtain a list of default virtualization storage classes by running the following command:

File: docs/runbooks/CDIMultipleDefaultVirtStorageClasses.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+```bash
+$ kubectl get sc -o json | jq '.items[].metadata|select(.annotations."storageclass.kubevirt.io/is-default-virt-class"=="true")|.name'
+```
+
+## Mitigation
+
+Ensure the default virtualization storage class annotation is only present on a single storage class of your choice.
```
Comment (apinnick) [inline docs/runbooks/CDIMultipleDefaultVirtStorageClasses.md:24]: Suggestion: 

Ensure that only one storage class has the default virtualization storage class annotation.

**Review by apinnick — CHANGES_REQUESTED** [review]

**Comment by apinnick** [comment]
@akalenyu Is this alert for 4.14?

**Comment by akalenyu** [comment]
> @akalenyu Is this alert for 4.14?

4.15

**Comment by akalenyu** [comment]
@sradco @apinnick addressed comments

**Comment by sradco** [comment]
/lgtm

@apinnick Can I merge ?

**Review by sradco — APPROVED** [review]

**Comment by apinnick** [comment]
D/S runbook task for 4.15 - CNV-35292

**Pull request merged**